bin/truffleruby.sh

```diff
@@ -105,7 +105,7 @@ if [ $on_graalvm = false ]; then
                 graal_sdk="$root_parent/${repo_name}/sdk/mxbuild/dists/graal-sdk.jar"
             fi
         done
-        if [ -n "$source_truffle" -a -f "$source_truffle/dists/truffle-api.jar" ]; then # Source Truffle suite
+        if [ -n "$source_truffle" ] && [ -f "$source_truffle/dists/truffle-api.jar" ]; then # Source Truffle suite
```

Comment:
Interesting, I did not get a shellcheck warning for this one. Probably I have a older version.